gensim/models/coherencemodel.py

```diff
@@ -4,9 +4,9 @@
 # Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
-"""
-Module for calculating topic coherence in python. This is the implementation of
+"""Module for calculating topic coherence in python. This is the implementation of
 the four stage topic coherence pipeline from the paper [1]_.
+
```

Comment:
please use
```
`some text <http://...>`_
```
instead of `[1]_`, because `[num]` can't be parsed correctly with `autosummary` plugin (this related to #1809), here and everywhere